repaygithub/ui-tools
# CACTUS-379 :: Layout Components

Ticket: https://repayonline.atlassian.net/browse/CACTUS-379

~I ended up getting rid of the `Layout` template files and rendering all of the layout components in the main `App` file, but if you guys think it'd be better to abstract the layout pieces out into a separate component, I could do that too.~
The name of the layout component built into the app has been changed from `Layout` to `AppLayout` so as not to clash with the `Layout` from the component library.

There's no usage of `ActionBar.Panel` because that hasn't been released yet, but I think we should add an example using that when it does get released.

### Testing
1. Check out this branch and navigate to `modules/create-repay-ui`.
2. Run `yalc publish`
3. Run `yarn global add ~/.yalc/packages/@repay/create-ui/0.2.1`
4. In some other directory, run the `create-repay-ui` CLI
5. Run the generated app and make sure the layout pieces are working.
